src/app/modules/product/_service/category.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { Category } from '../_model/category';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CategoryService{
+
+  constructor() { }
+  
+  getCategories() : Category[]{
+    let categories : Category[] = [];
+    let category : Category =  new Category(0, "a", "a", 0); categories.push(category);
+    category =  new Category(1, "b", "b", 1); categories.push(category); 
+    category =  new Category(2, "c", "c", 0);categories.push(category);
```

Comment:
Otra manera de hacerlo es:
```suggestion
    categories.push(new Category(0, "a", "a", 0));
    categories.push(new Category(1, "b", "b", 1));
    categories.push(new Category(2, "c", "c", 0));
```

> No es necesario aceptar esta sugerencia. Se puede mantener como está sin problema.